api/src/main/java/org/openmrs/module/eptsreports/reporting/library/dimensions/TxNewAgeDimensionCohort.java

```diff
@@ -24,7 +24,8 @@
   @Override
   public Mapped<CohortDefinition> createXtoYAgeCohort(String name, Integer minAge, Integer maxAge) {
     return EptsReportUtils.map(
-        genericCohorts.getAgeOnArtStartDate(minAge, maxAge, true), "location=${location}");
+        genericCohorts.getAgeOnArtStartDate(minAge, maxAge, true),
+        "onOrAfter=${onOrAfter},onOrBefore=${onOrBefore},location=${location}");
```

Comment:
Are these mappings suppossed to follow this https://github.com/jembi/openmrs-module-eptsreports/blob/develop/api/src/main/java/org/openmrs/module/eptsreports/reporting/library/dimensions/TxNewAgeDimensionCohort.java#L37-L41